server/test/controllers/admin/AdminExportControllerTest.java

```diff
@@ -100,7 +101,7 @@ public void hxExportProgram_invalidProgramId_badRequest() {
     Result result =
         controller.hxExportProgram(
             addCSRFToken(
```

Comment:
**question (non-blocking):** Do we still want to add the CSRFToken here?

Reply:
Coming in https://github.com/civiform/civiform/pull/8075 :) https://github.com/civiform/civiform/issues/8046 has a list of all the related PRs.